features/support/hooks.js

```diff
@@ -37,33 +36,31 @@ module.exports = function () {
     // application server mock
     this.app = makeExpressApp(this)
     this._server = http.createServer(this.app)
-    var primusOpts = {
+    const primusOpts = {
```

Comment:
> there's a difference between "const" and "a constant"

-- my roommate.

I totally get where you're coming from though. What I'm asserting here with the `const` is that while in this function, I should _never_ be re-assigning it. This prevents me from doing silly things, for sure. I TOTALLY agree that constants should go to the top, but this is a language semantic that I am using to prevent silly mistakes (linters pick up on this too - it's awesome)